TypeCobol/Compiler/Sql/Model/SavePointClause.cs

```diff
@@ -5,8 +5,8 @@ namespace TypeCobol.Compiler.Sql.Model
 {
     public class SavePointClause : SqlObject
     {
-        public SymbolReference SavePointName { get; }
-        public SavePointClause(SymbolReference savePointName)
+        public SymbolDefinition SavePointName { get; }
```

Comment:
I searched for the difference between SymbolDefinition and SymbolReference , and i think only savepointName and sqlColumnName will be changed to SymbolDef .For TableOrViewOrCorrelationName it could be qualifiedSymbolRef so i maintained it SymbolRef .